equinor/EchoComponents
# chore!: #60957 - Update eds

## Prerequisites

Please make sure you can check the following boxes:


## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


### Description

Upgrading to latest EDS core.
As I saw no breaking changes affect this repo, but pls do double check - maybe I missed something.

More frequently used components by Echo which are affected by breaking changes:
- Scrim (not used in components)
- Search (not used in components)
- Menu (not used in components)

More details in the EDS Changelog:
https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/CHANGELOG.md

### How to test
`npm run storybook` => check if any components in story book behave or look weird

### Related PRs
- EchopediaWeb:
https://github.com/equinor/EchopediaWeb/pull/1078
- EchoEvents:
https://github.com/equinor/echo-events/pull/40
- EchoFramework:
https://github.com/equinor/EchoFramework/pull/124
